java/code/src/com/redhat/rhn/manager/user/UserManager.java

```diff
@@ -735,6 +736,20 @@ public static boolean hasRole(Long uid, Role label) {
         return UserFactory.lookupById(uid).hasRole(label);
     }
 
+    /**
+     * Check RBAC memberships of the specified user.
+     * @param uid The id of the user to lookup.
+     * @param group the access group to check.
+     * @return {@code True} if the user is a member of the access group.
+     */
+    public static boolean isMemberOf(Long uid, AccessGroup group) {
```

Comment:
In this file you should also check the method "verifyChannelAdmin", which I think is still using the old roles.